### PR TITLE
Copy edit of the Orchestrator readme

### PR DESCRIPTION
**Pull request by JonathanFingold**
## Proposed Changes
Edit for style and clarity.

## Testing
Visual review of updated .md content.

### CODE REVIEW

**Comment by JonathanFingold** [comment]
@scheyal This is ready for your review. I included additional commentary in HTML comments, which should be removed before this is merged.

File: Orchestrator/README.md
Comment (scheyal) [inline Orchestrator/README.md:27]: Orchestrator is still PREVIEW. We should plan to doc in MS Docs by GA ~BUILD timeframe. Not now.

File: Orchestrator/README.md
Comment (scheyal) [inline Orchestrator/README.md:13]: They used the term arbitrate intentionally. Dependening on your perspective, it essentially selects the "correct" subsequent service to service the request. It doesn't really combine, rather selects a winner to route to.

File: Orchestrator/README.md
Comment (scheyal) [inline Orchestrator/README.md:59]: I agree, now that we have the other doc, we can trim this section, and refer to the other doc.

File: Orchestrator/README.md
Comment (scheyal) [inline Orchestrator/README.md:158]: Change  API reference to: https://aka.ms/bforchestratorapi
(just added that)

File: Orchestrator/README.md
Comment (scheyal) [reply Orchestrator/README.md:59]: I'll leave it up to you to decide what's best here :)

**Review by scheyal — APPROVED** [review]
See comments.

File: Orchestrator/README.md
Comment (JonathanFingold) [reply Orchestrator/README.md]: Yes, please add a prerequisites note to the other article. (I'm keeping a similar note in this article.) I'll call this good for now.

File: Orchestrator/README.md
Comment (JonathanFingold) [reply Orchestrator/README.md]: fixed

File: Orchestrator/README.md
Comment (JonathanFingold) [reply Orchestrator/README.md]: done

**Review by scheyal — COMMENTED** [review]
Good cleanup :)

**Pull request merged**
